django_prometheus/middleware.py

```diff
@@ -43,6 +43,10 @@ def process_response(self, request, response):
 requests_latency = Histogram(
     'django_http_requests_latency_seconds',
     'Histogram of requests processing time.')
+requests_latency_by_view_and_method = Histogram(
```

Comment:
I believe this metric obsoletes of django_http_requests_latency_second. Is that the case? If so we should remove the less specific one (and release a major version as this breaks metric compatibility)